JS/see_article.js

```diff
@@ -172,8 +192,25 @@ function create_see_article(article) {
         }
         if (article_already_in_cart == false) {
             tab.push({ 'id': article.id, 'name': article.name, 'price': article.price, 'brand': article.brand, 'description': article.description, 'preview': article.preview, 'quantity': quantity_article });
+            document.getElementById("nb_article_in_cart").textContent = tab.length;
         }
         sessionStorage.setItem('articleToCart2', JSON.stringify(tab));
+        //Soit on affichera directement la pannier pour indiquer l'ajout soit on devra avoir un pop up
+       // location.href = "cart.html";
+
+        document.getElementById("overlay_alert").style.display = "block";
+        document.getElementById("popupAlert").style.display = "block";
+        myVar = setTimeout(alertClosed, 1500);
```

Comment:
there is no need to keep the timeout in a variable as it isn't used.

Reply:
Corrected.